meson.build

```diff
@@ -104,20 +104,18 @@ if isWindows
 	# if MSVC, b_coverage will always return false because
 	# Meson doesn't officially support coverage
 	if cxx.get_id() == 'msvc' and get_option('buildtype') == 'debug'
-		coverageRunner = find_program('OpenCppCoverage', required: false)
-
-		if coverageRunner.found()
-			message('Enabling coverage under MSVC')
-			coverageArgs = [
-				'--sources', '@0@\\impl\\*'.format(meson.current_source_dir()),
-				'--sources', '@0@\\substrate\\*'.format(meson.current_source_dir()),
-				'--sources', '@0@\\test\\*'.format(meson.current_source_dir()),
-				'--sources', '@0@\\*'.format(meson.current_build_dir()),
-				'--modules', meson.current_build_dir(),
-				'--export_type'
-			]
-			coverage = true
-		endif
+		coverageRunner = find_program('OpenCppCoverage', required: true)
```

Comment:
Did you mean to make MSVC debug builds require OpenCppCoverage?

Reply:
Yes, as a safety measure-- there was *no* warning *at all* that Codecov couldn't find anything to ship. And that is in turn not an error for Codecov 🤦‍♀️

Reply:
Ahhh, that makes sense. Shame that Meson turning b_coverage into an always-false value on Windows causes so many extra problems!

Perhaps it might be an idea to document this, but given we can refer back to this PR for the reasoning now, we'll accept this and merge.